pull.go

```diff
@@ -112,8 +112,11 @@ func (c *Client) fetch(ctx context.Context, rCtx *RemoteContext, ref string, lim
 		childrenHandler := images.ChildrenHandler(store)
 		// Set any children labels for that content
 		childrenHandler = images.SetChildrenLabels(store, childrenHandler)
-		// Filter children by platforms
-		childrenHandler = images.FilterPlatforms(childrenHandler, rCtx.PlatformMatcher)
+		// Filter non-image manifest's children
+		childrenHandler = remotes.FilterInvalidManifestHandler(childrenHandler)
```

Comment:
Can you remove this change from this PR and maybe we can discuss in an issue. The topic is "how should pull handle unknown manifests". I think we should error on the side of they are pulled into the content store, but then nothing happens. Perhaps this should be gated on an option, but as is this is too opinionated.

Reply:
sure. will do.